### ISSUE TITLE
Use built-in version detection

### ISSUE THREAD

**Issue by jonom** [bluehousegroup/silverstripe-upgrade-notification#3]
You can use [LeftAndMain::CMSVersion()](http://api.silverstripe.org/3.1/class-LeftAndMain.html#_CMSVersion) to get the software version. This is safer than relying only on composer.lock as it may not exist on a live site.

**Comment by andrewvt** [comment]
Good call, I'll try it and see if it is reliable enough.  Hadn't spotted that function.

Interestingly the silverstripe_version file in most versions is actually empty and the  LeftAndMain::CMSVersion() function actually tries the composer.lock file first and silverstripe_version as a fallback.  So in the end it accomplishes the same thing.  I agree using the built in function will be better than maintaining my own assuming it is reliable through all versions.

**Comment by jonom** [comment]
Yeah may not be as reliable for older versions of the framework. I wouldn't have known about this function either but one day my mouse was hovering over the SS logo in the CMS and it brought up the version number. So I figured it had to be somewhere :P
